app/components/button/index.tsx

```diff
@@ -48,12 +54,15 @@ const Button = ({
             onPress={onPress}
             testID={testID}
         >
-            <Text
-                style={txtStyle}
-                numberOfLines={1}
-            >
-                {text}
-            </Text>
+            <View style={styles.container}>
+                {compassIcon}
```

Comment:
A color has to be passed to the compass icon, that comes form the txt style. Shouldn't it be better to pass the compass icon name and do the formatting here?

Reply:
I thought about it but then, we would need to set the name, color and size, that's why I went with this, but I can take another look and see what I can do.

Reply:
The size could be added also here, based on the text size. But 0/5.

Worst case scenario, we can leave it as it is now, and revisit later.